__mocks__/cognito-fake.js

```diff
@@ -0,0 +1,18 @@
+export default class CognitoFake {
+  listUsers() {
+    return {
+      promise: async () => {
```

Comment:
Yes, this is gross, but this is the interface of the dependency we're faking so `/shrug`!

Reply:
I wouldn't mind having your PR comment be in the actual code ...